src/views/ExportSuccess.vue

```diff
@@ -26,12 +26,18 @@ import KeyguardClient from '@nimiq/keyguard-client';
 @Component({components: {SmallPage, Loader}})
 export default class ExportSuccess extends Vue {
     @Static private request!: ParsedSimpleRequest;
+    @Static private originalRouteName?: string;
     @State private keyguardResult!: KeyguardClient.ExportResult;
 
     private loaderState = Loader.State.LOADING;
     private successMessage = '';
 
     private async mounted() {
+        if (this.originalRouteName || this.request.kind === RequestType.MIGRATE) {
+            this.$rpc.routerReplace(RequestType.MIGRATE);
```

Comment:
I know this is cumbersome, but _ideally_ we would remember which accounts have been successfully exported, to already set the `wordsExported` flags accordingly after migration.